src-tauri/icons/generate_icons.py

```
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+"""
+Generate all required icon sizes for Tauri from a single icon.png
+Requires: pip install Pillow
+"""
+
+import os
+import sys
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)